src/integrationTest/resources/application.yml

```diff
@@ -22,7 +22,7 @@ spring:
         registration:
           oidc:
             client-id: microservicekey-rd-professional-api
-            client-secret: ${S2S_SECRET:K6KA6PRLRFWQLSD6}
+            client-secret: ${S2S_SECRET:AAAAAAAAAAAAAAAA}
```

Comment:
We can also remove client.secret which is used for sidam bearer token

Reply:
where it is  used ?

Reply:
it is masked  now to some value